### PR TITLE
Added Gitpod Button

### PR DESCRIPTION
**Pull request by Sandared**
I added a Gitpod Button to Readme.md. 

Now all a user has to do to run the quickstart is to 
* open Gitpod
* cd to examples/quickstart
* mvn verify
* run the jar. 

Even a preview is given within Gitpod
Further enhancements could be to add a gitpod config file to the repository that already contains thos steps as setup command and exposes port 8080 by default. This way a user directly lands in a wowrkspace where everything is running and he can play araound with the code.

Signed-off-by: Thomas Driessen <thomas.driessen.td@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <h1><img src="http://enroute.osgi.org/img/enroute-logo-64.png" witdh=40px style="float:left;margin: 0 1em 1em 0;width:40px">
 OSGi enRoute</h1>
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io#https://github.com/Sandared/jmh-benchmarks/blob/master/benchmarks/src/main/java/io/jatoms/jmh/OSGiBenchmark2.java)
```
Comment (bjhargrave) [inline README.md]: 1) Gitpod is a for-profit company. I am not sure why OSGi would want to promote them.

2) The referenced URL goes to your github account and I don't see that OSGi would want to link to that.

File: README.md [hunk 2]
```diff
@@ -9,7 +11,7 @@ This repository contains the code for the enRoute tutorials, and also defines us
 
 ## Contributing
 
-Want to contribite to osgi.enroute? See [CONTRIBUTING.md](CONTRIBUTING.md) for information on building, testing and contributing changes.
+Want to contribute to osgi.enroute? See [CONTRIBUTING.md](CONTRIBUTING.md) for information on building, testing and contributing changes.
```
Comment (bjhargrave) [inline README.md]: Perhaps make this fix in a separate PR so we can merge independent of any Gitpod discussion.

Comment (Sandared) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (Sandared) [reply README.md]: 1) True, but just to onboard users quickly and letting them play around wih the quickstarter app the 10 free hours are more than sufficient I think. I personally would love to just click a button and see the examples working and being able to play around with the code, but if thats interfering with OSGi-Alliance guidelines then of course it's a no-go. Didn't think about that before.
2) Sorry, Copy/Paste error. Will fix that.

**Comment by Sandared** [comment]
I'm not sure why appveyor build failed. According to the log it just stopped after downloading the first maven dependency.

File: README.md (hunk 1 above)
Comment (svenefftinge) [reply README.md]: Re: 1) Gitpod is now free for open-source projects (see https://www.gitpod.io).
